### ISSUE TITLE
Token silent renewal fails on Chrome only

### ISSUE THREAD

**Issue by lecaillon** [AzureAD/microsoft-authentication-library-for-js#303]
### Context

After calling `loginRedirect` the call to `acquireTokenSilent` fails and `acquireTokenRedirect` is executed causing the entire application to reload.

### Trace

> BSSO Telemetry: {"nonce":"**********","type":"ChromeSsoTelemetry","traces":["BrowserSSO Initialized","Creating chrome provider","Channel message[CreateProviderAsync] received","Error: NoExtension: Extension is not installed.","SSO cookie detected. Refreshing page."]}

> A silent sign-in request was sent but no user is signed in. The cookies used to represent the user's session were not sent in the request to Azure AD. This can happen if the user is using Internet Explorer or Edge, and the web app sending the silent sign-in request is in different IE security zone than the Azure AD endpoint (login.microsoftonline.com)

This behavior is specific to Chrome. The same code works fine in Firefox and IE and Edge. 

### Cause (and solution)

**It appears it was my flash and ads blocker extensions that caused this behavior. I suspect those extensions messed with the AAD cookie. So deactivate them resolved my problem.**

**Comment by kaburkett** [comment]
I am receiving a similar error.

```BSSO Telemetry: {"result":"Error", "error":"NoExtension","type":"ChromeSsoTelemetry","data":{},"traces":["BrowserSSO Initialized","Creating ChromeBrowserCore provider", "Sending message for method CreateProviderAsync","Received message for method CreateProviderAsync","Error: ChromeBrowserCore error NoExtension: Extension is not installed."]}```

The only extension I have installed is Redux DevTools. This may be caused by any chrome extension at all.

**Comment by emadalsous** [comment]
I got this error too for one of my project users. In my case, it was related to a cookies security configuration in the user's browser. I had to go to Settings --> Privacy and security --> Site settings --> Cookies and site data --> set Block third-party cookies to off.
If Block third-party cookies is on, it will block third party cookies, the authorization server can't be accessed in the application.

I hope this could help someone.

Best regards,
Emad
